### PR TITLE
shell-loop.sh 

### PR DESCRIPTION
**Pull request by maksudatoma**
2nd part of HW2

### CODE REVIEW

**Review by showmic09 — CHANGES_REQUESTED** [review]
@maksudatoma Please add the file inside your directory. So from the terminal move the file to Toma/HW2. Make sure that you are on the HW2-Toma-patch01 branch and do the following
- git add <filename> and afterwards
- git commit -m "Changed the file's location" and finally
- git push origin HW2-Toma-patch01

**Comment by maksudatoma** [comment]
@showmic09 I have changed the file's location.

**Review by showmic09 — CHANGES_REQUESTED** [review]
Hi @maksudatoma if you look under Files changed you will see that you have two files. Can you please delete the other file that's not in the folder. If you want you can use GitHub website for this. otherwise the command from terminal is 
- git rm `<file name>`

File: shell-loop.sh [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```
Comment (showmic09) [inline shell-loop.sh:1]: Please delete this file.

File: shell-loop.sh
Comment (showmic09) [inline shell-loop.sh:1]: Please delete this file @maksudatoma

**Review by showmic09 — APPROVED** [review]

File: shell-loop.sh
Comment (showmic09) [reply shell-loop.sh:1]: LGTM now. Thanks

**Pull request merged**